mlx/robot2rst.mako

```diff
@@ -59,7 +63,7 @@ filtered_tags = [tag for tag in test.tags if re.search(tag_regex, tag)]
 % endfor
 
 % if str(test.doc):
-${generate_body(str(test.doc))}
+${generate_body(str(test.doc).strip())}
```

Comment:
Why would you want to strip stuff here?

Reply:
It's cleaner to move this inside the function. Done in 76d905bcb1ec37f8f760b63555b2182c5937e0f7